crouton.php

```diff
@@ -427,6 +427,10 @@ public function adminOptionsPage()
                                 <label for="root_server">Default Root Server: </label>
                                 <input id="root_server" type="text" size="50" name="root_server" value="<?php echo $this->options['root_server']; ?>" /> <?php echo $connect; ?>
                             </li>
+                            <li>
+                                <input type="checkbox" id="use_tomato" name="use_tomato" value="1"/>
+                                <label for="use_tomato">Use Tomato &#127813; (Root Server Aggregator)</label>
```

Comment:
Might be helpful to have a help tooltip explaining what 🍅 is.

Reply:
true true